tilequeue/command.py

```diff
@@ -1047,6 +1047,28 @@ def tilequeue_initial_load_wof_neighbourhoods(cfg, peripherals):
     logger.info('Loading ... done')
 
 
+def tilequeue_dump_tiles_of_interest(cfg, peripherals):
+    logger = make_logger(cfg, 'dump_tiles_of_interest')
+    logger.info('Dumping tiles of interest')
+
+    logger.info('Fetching tiles of interest ...')
+    coords = peripherals.redis_cache_index.fetch_tiles_of_interest()
+    n_toi = len(coords)
+    logger.info('Fetching tiles of interest ... done')
+
+    toi_filename = "toi.txt"
+
+    logger.info('Writing %d tiles of interest to %s ...', n_toi, toi_filename)
+    with open(toi_filename, "w") as f:
+        for coord in map(coord_unmarshall_int, coords):
```

Comment:
It would be better to use either `imap` here or call the function during the iteration.